app/controllers/offices_controller.rb

```diff
@@ -0,0 +1,24 @@
+class OfficesController < ApplicationController
+  def index
+    return unless search
+    return @error_message = search.error_message unless search.valid?
+
+    @result = search.result
+    @transportation_offices = TransportationOffice.by_distance_with_shipping_office(latitude: @result[:latitude], longitude: @result[:longitude]).paginate(page: params[:page])
+  end
+
+  private
+
+  def search
+    return coordinates_search if params[:coordinates].present?
+    return zip_code_search if params[:postal_code].present?
```

Comment:
Why use both "zip code" and "postal code"? Gets a bit confusing

Reply:
Agreed that it's confusing. I named the field in the database `postal_code` thinking maybe there would be non-ZIP international postal codes for overseas locations (ZIP being a US-specific standard). I'm not entirely sure that non-ZIP postal codes will be a thing, though…

@jamesatheyDDS, given what you've seen regarding installations and addresses, what do you think? Should we standardize naming as `postal_code`, `zip_code`, or something else?

Reply:
I originally did not worry about international post codes for a very specific reason - all military bases, even those abroad, have standard US ZIP Codes.

Reply:
so...i wonder, does this account for folks living off base abroad? I realize that's probably a small number, but still

Reply:
> does this account for folks living off base abroad

Where this naming convention _might_ come in to play is if we add locations of off-base services in countries outside the United States (e.g. public weight scales). I don't want to belabor the naming, though. If it's easier for everyone, I can rename that column to `zip_code` for consistency with the UI. If we find we need to change it later, we can.